specification/syntex/resource-manager/Microsoft.Syntex/preview/2022-09-15-preview/syntex.json

```diff
@@ -0,0 +1,493 @@
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": false,
+          "description": "Document processor properties.",
+          "$ref": "#/definitions/DocumentProcessorProperties"
+        }
+      }
+    },
+    "DocumentProcessorPatchableProperties": {
+      "type": "object",
+      "description": "Document processor properties that can be patched",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "description": "Resource tags.",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "properties": {
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "description": "Syntex properties for PATCH.",
+          "properties": {
+            "spoTenantUrl": {
+              "type": "string",
+              "description": "The URL of an SharePoint Online (SPO) tenant associated with this document processor resource"
+            },
+            "spoTenantId": {
+              "type": "string",
+              "description": "The Id (GUID) of an SharePoint Online (SPO) tenant associated with this document processor resource"
+            }
+          }
+        }
+      }
+    },
+    "DocumentProcessorProperties": {
+      "type": "object",
+      "description": "Document processor properties",
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "The managed resource provisioning state."
+        },
+        "spoTenantUrl": {
+          "type": "string",
+          "description": "The URL of an SharePoint Online (SPO) tenant associated with this document processor resource"
+        },
+        "spoTenantId": {
+          "type": "string",
+          "description": "The Id (GUID) of an SharePoint Online (SPO) tenant associated with this document processor resource"
+        }
+      },
+      "required": [
+        "spoTenantUrl",
+        "spoTenantId"
+      ]
+    },
+    "ProvisioningState": {
+      "description": "Provisioning status of the managed resource.",
+      "readOnly": true,
+      "enum": [
+        "Succeeded",
+        "Failed",
+        "Canceled"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": true
+      }
+    },
+    "DocumentProcessorList": {
+      "type": "object",
+      "description": "Document processor list",
+      "properties": {
+        "value": {
+          "description": "Array of document processors.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DocumentProcessorResource"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ],
```

Comment:
Are document processor resource names globally unique?  If not it would probably be better not to specify 'name' so you get the default of 'id', which would be globally unique

Reply:
Thank you, good point, it is not globally unique.